components/Channel/interfaces.rst

```diff
@@ -0,0 +1,72 @@
+Interfaces
+==========
+
+Model Interfaces
+----------------
+
+.. _component_channel_model_channel-interface:
+
+ChannelInterface
+~~~~~~~~~~~~~~~~
+
+This interface should be implemented by every custom sale channel model.
+
+.. note::
+   This interface extends :ref:`component_resource_model_timestampable-interface`. |br|
+   For more detailed information go to `Sylius API ChannelInterface`_.
+
+.. _Sylius API ChannelInterface: http://api.sylius.org/Sylius/Component/Channel/Model/ChannelInterface.html
+
+.. _component_channel_model_channel-aware-interface:
+
+ChannelAwareInterface
+~~~~~~~~~~~~~~~~~~~~~
+
+This interface should be implemented by models associated
+with a specific sale channel.
+
+.. note::
+   For more detailed information go to `Sylius API ChannelAwareInterface`_.
+
+.. _Sylius API ChannelAwareInterface: http://api.sylius.org/Sylius/Component/Channel/Model/ChannelAwareInterface.html
+
+Service Interfaces
+------------------
+
+.. _component_channel_model_channels-aware-interface:
+
+ChannelsAwareInterface
+~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Actually, this is a model interface too. It is same as ChannelAwareInterface, it is for models associated with multiple channels. :)